### ISSUE TITLE
no rule to make Clipper2

### ISSUE THREAD

**Issue by briansturgill** [elalish/manifold#703]
I am unable to build.

**Comment by pca006132** [comment]
what error do you get? did you try a clean build?

**Comment by briansturgill** [comment]
manifold/CMakeFiles/manifold.dir/src/sort.cpp.o -MF CMakeFiles/manifold.dir/src/sort.cpp.o.d -o CMakeFiles/manifold.dir/src/sort.cpp.o -c /home/brian/work/manifold/manifold/src/manifold/src/sort.cpp
make[2]: *** No rule to make target 'Clipper2', needed by 'src/manifold/libmanifold.so.2.3.1'.  Stop.
make[2]: Leaving directory '/home/brian/work/manifold/manifold/build'
make[1]: *** [CMakeFiles/Makefile2:584: src/manifold/CMakeFiles/manifold.dir/all] Error 2
make[1]: Leaving directory '/home/brian/work/manifold/manifold/build'
make: *** [Makefile:139: all] Error 2
./build_manifold: line 7: test/manifold_test: No such file or directory

**Comment by briansturgill** [comment]
build_manifold is a shell script that I use to git and buiild the latest and greatest...
```sh
rm -rf manifold
git clone --recurse-submodules https://github.com/elalish/manifold.git
cd manifold
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release -DMANIFOLD_JSBIND=OFF -DMANIFOLD_CBIND=ON -DMANIFOLD_PYBIND=ON -DMANIFOLD_EXPORT=OFF -DBUILD_SHARED_LIBS=ON -DMANIFOLD_TEST=ON .. && make
test/manifold_test
```

**Comment by briansturgill** [comment]
@pca006132 manifold/src/third_party has quick_hull but does not have clipper2. This is true on github when I
look there as well. It's like the submodule got removed.

**Comment by pca006132** [comment]
yes I removed it, but it should get the dependency automatically

can you attach the full cmake output?

**Comment by briansturgill** [comment]
@pca006132 Here's the output:
[out.txt](https://github.com/elalish/manifold/files/13934309/out.txt)

**Comment by pca006132** [comment]
It is weird, `pkg-config` found clipper2 but somehow make is trying to make `Clipper2` target. What distribution are you using and did you run anything like `cmake --install .` before?

**Comment by pca006132** [comment]
can you try to add the following line to https://github.com/elalish/manifold/blob/master/manifoldDeps.cmake#L11-L14
```
message("Clipper2_LINK_LIBRARIES: ${Clipper2_LINK_LIBRARIES}")
message("Clipper2_INCLUDE_DIRS: ${Clipper2_INCLUDE_DIRS}")
```

and show me the output?

**Comment by briansturgill** [comment]
Clipper2_LINK_LIBRARIES: 
Clipper2_INCLUDE_DIRS: /usr/local/include

As far as I can tell I don't have clipper2 installed at all...
For some reason I did have clipper2 header files (no libs) in /usr/local/lib
I deleted them.

Now both messages show empty.

Still failing, doing global find for -iname *clipper2*
Ah found /usr/local/lib/pkgconfig/Clipper2.pc.

Removed it.

That did it... strangely I don't seem to have an actual library installed for Clipper2!

Thanks much for the help!
